library/google-storage/googleStorageService.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const bucketName = 'bucket-name-sxq3f';
+
+const getFilePath = (namespace, key) => namespace + "/" + key;
+
+const put = async (googleStorage, namespace, key, value) => {
+  let filePath = getFilePath(namespace, key);
+  var tmp = require('tmp');
```

Comment:
move `require` to top of the file.
also, don't use var. use `let` or `const` (in this case `const`)